classification/helper_functions.py

```diff
@@ -0,0 +1,91 @@
+from IPython.display import Markdown, display
+import pandas as pd
+import numpy as np
+np.random.seed(42)
+rng = np.random.default_rng(42)
+import matplotlib.pyplot as plt
+import seaborn as sns
+from imblearn.over_sampling import SMOTE
+from imblearn.under_sampling import RandomUnderSampler
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier
+from xgboost import XGBClassifier
+from sklearn.model_selection import GridSearchCV
+from sklearn.metrics import roc_auc_score, roc_curve
+
+def grid_train_random_forest(X, y, params, n_folds, eval_metric):
+    """
+    Train Random Forest binary classifier using a grid of hyperparameters. Return
+    the best model according to the specified metric.
+
+    Args:
+        X: Array-like of shape (n_samples,n_features) - Test feature data.
+        y: Array-like of shape (n_samples,) - Test target data.
+        params: Dictionary - Parameter grid on which to perform cross validation.
+        n_folds: int - Number of folds to use for cross validation.
+        eval_metric: str - Metric to use for evaluating model performance in cross validation.
+
+    Returns:
+        model: Best Random Forest model according to evaluation metric.
+
+    Examples:
+        model = grid_train_random_forest(X, y, params, 4, "accuracy")
+    """
+
+    # TODO: Implement this function
+def grid_train_random_forest(X, y, params, n_folds = 4, eval_metric = 'accuracy'):
+    random_search = GridSearchCV(RandomForestClassifier(), params, cv=n_folds, scoring=eval_metric)
+    random_search.fit(X, y)
+    model = random_search.best_estimator_
```

Comment:
Need to return this value (I think python returns the last value referenced/set by default but it's good practice to always manually return something if the function calls for it)